sdk/communication/Azure.Communication.Identity/tests/SessionRecords/CommunicationIdentityClientLiveTests/GettingTokenWithMultipleScopesWithConnectionString.json

```diff
@@ -23,32 +23,33 @@
       "ResponseHeaders": {
         "api-supported-versions": "2020-07-20-preview2, 2021-03-07",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Wed, 17 Feb 2021 00:51:49 GMT",
-        "MS-CV": "rgf1oJQcJUqATgiBVl\u002B5Mg.0",
+        "Date": "Fri, 19 Feb 2021 01:16:51 GMT",
+        "MS-CV": "rDtfMqaTekSMOJ7sBiC0/A.0",
+        "Request-Context": "appId=",
         "Strict-Transport-Security": "max-age=2592000",
         "Transfer-Encoding": "chunked",
-        "X-Azure-Ref": "0pmgsYAAAAABJQ3tklez7QK099Ro9kInLWVZSMzBFREdFMDMxMQA5ZmM3YjUxOS1hOGNjLTRmODktOTM1ZS1jOTE0OGFlMDllODE=",
+        "X-Azure-Ref": "0hBEvYAAAAADSrmyxOnLYS7XCJvKog3FnWVZSMzBFREdFMDQwOQA5ZmM3YjUxOS1hOGNjLTRmODktOTM1ZS1jOTE0OGFlMDllODE=",
         "x-ms-client-request-id": "5bcf0bff02c0de827e06967978ceaa22",
-        "X-Processing-Time": "16ms"
+        "X-Processing-Time": "9ms"
       },
       "ResponseBody": {
         "identity": {
-          "id": "Sanitized"
+          "id": "8:acs:cb58c3bb-a8ef-4e54-8c12-4a8e652eba58_00000008-57be-0fd2-6a0b-343a0d005880"
         }
       }
     },
     {
-      "RequestUri": "https://sanitized.communication.azure.com/identities/sanitized/:issueAccessToken?api-version=2021-03-07",
+      "RequestUri": "https://chitchat-test.communication.azure.com/identities/8%3Aacs%3Acb58c3bb-a8ef-4e54-8c12-4a8e652eba58_00000008-57be-0fd2-6a0b-343a0d005880/:issueAccessToken?api-version=2021-03-07",
```

Comment:
Is there a reason why we used to sanitize user ids in some places but not in others?

Reply:
use ids are not to be sanitized at all. Where did you see it being sanitized?